packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -288,7 +290,7 @@ function _Autocomplete<ItemType>(
         id={menuProps.id}
       >
         <Popover.Trigger>
-          <div className={styles.combobox}>
+          <div {...comboboxProps} className={styles.combobox}>
```

Comment:
is this change relevant?

Reply:
It was part of the v7 bump, I am reverting everything back from v6.